### DISCUSSION TITLE
Properties of type enum not included in the .received file

### DISCUSSION THREAD

**Discussion by razvan-bugheanu-bzy** [VerifyTests/Verify#952] [Q&A]
Hi,

I've been using the library with some objects of mine that include properties of enum type. To my surprise, they are not included in the .received file. I've tried with both Argon and UseStrictJson() options and got same outcome. Looking over test samples in the repo, I found this one, which reproduces the same issue I have:

https://github.com/VerifyTests/Verify/blob/0c0277ec1d2e8b75fb1b71a95cb7de0db97499ad/src/SortedPropertiesTests/SortedPropertiesTests.cs

Can someone say if there is a way to include enum type properties in the .received file ?

Thanks,

**Comment by SimonCropp** [comment]
try this 
```
static class ModuleInitializer
{
    [ModuleInitializer]
    public static void Init() =>
        VerifierSettings.AddExtraSettings(_ => _.DefaultValueHandling = DefaultValueHandling.Include);
}
```

**Reply by razvan-bugheanu-bzy** [reply]
It worked. As a side effect, it also added defaults for all properties defined in the class (e.g. double -> 0.0), which is not necessarily something I was looking after. But if this is the only way, then it is better than nothing.

Thank you.

**Reply by SimonCropp** [reply]
happy to accept a pull request that adds a “DontScrubDefaultEnumValues” feature

**Reply by mattiasnordqvist** [reply]
Would you accept a PR that changes defaults to not scrub default values at all? Big breaking change but for the better in my opinion. Zeroes and nulls are as important values as any other.

**Reply by SimonCropp** [reply]
why. you can change the default globally for your project

**Reply by mattiasnordqvist** [reply]
Because I think it is a bad default. :)

**Reply by SimonCropp** [reply]
@mattiasnordqvist in that case. no
